lib/hdcontroller.js

```diff
@@ -328,11 +328,11 @@ class HDProxydClient {
                     // Drain the stream
                     res.resume();
                     res.on('end', () => {
-                        callback(err);
+                        done(err);
```

Comment:
can't you just pass the function `done` instead of using an anonymous function then calling `done` function ?